services/summaries/summaries.py

```diff
@@ -0,0 +1,194 @@
+import os
+import re
+import sys
+import time
+import json
+import logging
+import traceback
+import emoji
+import random
+import datetime
+from arango import ArangoClient
+
+DB_USER = os.getenv("DB_USER")
+DB_PASS = os.getenv("DB_PASS")
+DB_PORT = os.getenv("DB_PORT")
+DB_NAME = os.getenv("DB_NAME")
+DB_HOST = os.getenv("DB_HOST")
+
+SCAN_TYPES = ["https", "ssl", "dkim", "spf", "dmarc"]
+CHARTS = {"mail": ["dmarc", "spf", "dkim"], "web": ["https", "ssl"]}
+
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+
+
+def update_scan_summaries(host=DB_HOST, name=DB_NAME, user=DB_USER, password=DB_PASS, port=DB_PORT):
+    logging.info(f"Updating scan summaries...")
+
+    # Establish DB connection
+    connection_string = f"http://{host}:{port}"
+    client = ArangoClient(hosts=connection_string)
+    db = client.db(name, username=user, password=password)
+
+    if not db.has_collection("scanSummaries"):
+        db.create_collection(
+            "scanSummaries",
+            replication_factor=3,
+            shard_count=6,
+            write_concern=1,
+        )
+
+    for scan_type in SCAN_TYPES:
+        scan_pass = 0
+        scan_fail = 0
+        scan_total = 0
+        for domain in db.collection("domains"):
+            scan_total = scan_total + 1
+            if domain["status"][scan_type] == "fail":
+                scan_fail = scan_fail + 1
+            elif domain["status"][scan_type] == "pass":
+                scan_pass = scan_pass + 1
+
+        current_summary = db.collection("scanSummaries").get({"_key": scan_type})
+
+        summary_exists = current_summary is not None
+
+        if not summary_exists:
+            db.collection("scanSummaries").insert(
+                {
+                    "_key": scan_type,
+                    "pass": scan_pass,
+                    "fail": scan_fail,
+                    "total": scan_total,
+                }
+            )
+        else:
+            db.collection("scanSummaries").update_match(
+                {"_key": scan_type},
+                {"pass": scan_pass, "fail": scan_fail, "total": scan_total},
+            )
+
+        logging.info(f"{scan_type} scan summary updated.")
+
+    logging.info(f"Scan summary update completed.")
+
+
+def update_chart_summaries(host=DB_HOST, name=DB_NAME, user=DB_USER, password=DB_PASS, port=DB_PORT):
+    logging.info(f"Updating chart summaries...")
+
+    # Establish DB connection
+    connection_string = f"http://{host}:{port}"
+    client = ArangoClient(hosts=connection_string)
+    db = client.db(name, username=user, password=password)
+
+    if not db.has_collection("chartSummaries"):
+        db.create_collection(
```

Comment:
This needs to be removed for the database initilization script to be the so it does not override any settings that we could possibly change.